docs/user.md

```diff
@@ -0,0 +1,193 @@
+
+| Machine       | Count | CPU | RAM (GB) | Storage (GB) |
+| ------------- | ----- | --- | -------- | ------------ |
+| Bootstrap     | 1     | 4   | 16       | 100          |
+| Control Plane | 3     | 4   | 16       | 100          |
+| Compute       | 3     | 4   | 16       | 100          |
+| Dedicated Test| 1     | 4   | 8        | 100          |
+
+#### Environment Setup
+
+1. Choose one node with at least 8GiB of RAM and 4 vCPU
+2. Label node with `node-role.kubernetes.io/tests=""` (certification related pods will schedule to dedicated node)
+3. Taint node with `node-role.kubernetes.io/tests="":NoSchedule` (prevent other pods from running on dedicated node)
+
+*NOTE: certification pods will automatically have node-selectors and taint tolerations if you use the `--dedicated` flag.*
+
+There are two options to accomplish this type of setup...
+
+##### Option A: Command Line 
+
+```shell
+oc label node <node_name> node-role.kubernetes.io/tests=""
+oc taint node <node_name> node-role.kubernetes.io/tests="":NoSchedule
+```
+
+##### Option B: Machine Set 
+
+If you have support for OpenShift's Machine API then you can create a new `MachineSet` to configure the labels and taints. See [OpenShift documentation](https://docs.openshift.com/container-platform/latest/machine_management/creating-infrastructure-machinesets.html#binding-infra-node-workloads-using-taints-tolerations_creating-infrastructure-machinesets) on how to configure a new `MachineSet`. Note that at time of certification testing, OpenShift's product documentation may not mention your infrastructure provider yet! 
+
+Here is a `MachineSet` YAML snippet on how to configure the label and taint as well:
+
+```yaml
+      metadata:
+        labels:
+          openshift-tests: "true"
+          node-role.kubernetes.io/tests: ""
+      taints:
+        - key: node-role.kubernetes.io/tests
+          effect: NoSchedule
+```
+
+
+### Privilege Requirements
+
+A user with [cluster administrator privilege](https://docs.openshift.com/container-platform/latest/authentication/using-rbac.html#creating-cluster-admin_using-rbac) must be used with the provider certification tool. You also use the default kubeadmin user if you wish. 
+
+
+## Install
+
+
+### Prebuilt Binary
+
+The provider certification tool is shipped as a single executable binary which can be downloaded from:
+
+[https://github.com/redhat-openshift-ecosystem/provider-certification-tool](https://github.com/redhat-openshift-ecosystem/provider-certification-tool)
+
+The provider certification tool can be used from any system with network access to the OpenShift cluster under test. 
+
+
+### Build from Source
+
+See the development guide for more information.
+*TODO - link*
+
+
+## Usage
+
+
+### Run provider certification tests
+
+```sh
+openshift-provider-cert run 
+openshift-provider-cert run -w # Keep watch open until completion
+```
```

Comment:
@bostrt what do you think to split this block in something like this:

- Run the certification environment in the background:


```sh
openshift-provider-cert run
```

- Run the certification environment in the background and keep watching the progress:

```sh
openshift-provider-cert run -w
```